components/member-profile/contribution-type/index.js

```diff
@@ -14,8 +15,16 @@ const renderContributions = (contributions, fullName, imageLink, devUser) =>
     />
   ));
 
+const renderActiveTasks = (tasks) => {
+  return tasks.map((task, index) => {
+    if (task.status === 'active') {
```

Comment:
why are we checking if the status is `active`? I thought the API was returning only Active tasks.

Reply:
Ok i thought it'd returned all the tasks of that member like even the completed ones... I'll check again

Reply:
Yes please check once. let me know if the API is not returning only `Active` tasks. Will get in touch with the Backend Engineer and get it changed.

Reply:
The status of the tasks returned is eother`assigned` or `active` . So should I remove the condition?